### DISCUSSION TITLE
"bambooCollisionBoxDisabled" >>> universal rule

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#64] [Ideas]
I brought this already on the table, just with Leaves blocks in mind. 
Now I re-think about it, and why not expanding it to disable the collision box of possible all blocks.

A possible powerful rule.

**Comment by wendavid552** [comment]
Well, could you provide any reasons or a list of blocks to disable the collision boxes? Since this rule is because go acrossing bamboo forests with random collision boxes is really annoying, why would you like it?

**Reply by ghost** [reply]
The same counts for Leaves, and in the end, could be fun to experiment with other blocks.

**Comment by 1024-byteeeee** [comment]
Not considering expanding this rule. 😄
